### PR TITLE
add dd.md

### PR DESCRIPTION
**Pull request by WANG-lp**
add `dd.md` to osx and linux

### CODE REVIEW

**Comment by markusthoemmes** [comment]
Any reason why you chose to put it in linux and osx seperately, when there is also the "common" section?

**Comment by waldyrious** [comment]
The only difference seems to be capitalization in `bs=4M` vs `bs=4m`. Is this significant?

By the way, shouldn't the last example be named "Benchmark drive write performance", to match the verb tense in the other examples?

**Comment by igorshubovych** [comment]
Could you please squash the commits?

**Comment by WANG-lp** [comment]
@igorshubovych I have squashed to only one commit.
@waldyrious I think this difference is important for a new `*nix` user. 
@markusthoemmes There are some minor difference between different OS, I know this is not wise, but I just want to give the new user a correct guide.

**Comment by felixonmars** [comment]
I would actually argue that "Wipe a disk by writing random data to it" is not really a common use case, and "Make a bootable usb drive from a iso file" may mislead users since it depends on whether the iso file itself supports that (keyword: isohybrid).

File: pages/linux/dd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# dd
+
+> convert and copy a file
+
+- Make a bootable usb drive from a isohybrid file(such like archlinux-xxx.iso)
```
Comment (igorshubovych) [inline pages/linux/dd.md]: Plz add space before bracket

File: pages/osx/dd.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# dd
+
+> convert and copy a file
+
+- Make a bootable usb drive from a isohybrid file(such like archlinux-xxx.iso)
```
Comment (igorshubovych) [inline pages/osx/dd.md]: Please add space before bracket

Comment (felixonmars) [inline pages/osx/dd.md]: Maybe it should be "an isohybrid" instead of "a isohybrid"?

**Comment by igorshubovych** [comment]
LGTM

**Comment by felixonmars** [comment]
:+1: Thanks :)

**Pull request merged**